app/subscriber/src/components/info/InfoShield.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { FaShield } from 'react-icons/fa6';
+import { Tooltip } from 'react-tooltip';
+import { Col } from 'tno-core';
+
+import * as styled from './styled';
+
+export const InfoShield: React.FC = () => {
+  return (
+    <styled.InfoShield className="info-shield">
+      <FaShield data-tooltip-id="info-shield" />
+      <Tooltip clickable className="react-tooltip" id="info-shield" place="top" delayHide={800}>
```

Comment:
add a few ms delay with `delayHide` so it doesn't disappear if you accidentally mouse off while moving to a link.